pysat/utils/_core.py

```diff
@@ -92,15 +101,15 @@ def scale_units(out_unit, in_unit):
     if in_key not in accepted_units.keys():
         raise ValueError('Unknown input unit {:}'.format(in_unit))
 
-    if out_key == 'm' or out_key == 'm/s' or in_key == 'm' or in_key == 'm/s':
+    if out_key in ['m', 'm/s', 'm-3'] or in_key in ['m', 'm/s', 'm-3']:
         if in_key != out_key:
             raise ValueError('Cannot scale {:s} and {:s}'.format(out_unit,
                                                                  in_unit))
         # Recast units as keys for the scales dictionary and ensure that
         # the format is consistent
         rkey = ''
         for rr in replace_str.keys():
-            if out_key.find(rr):
+            if out_key.find(rr) >= 0:
```

Comment:
Found the bug.  The previous line was always True, so whatever was last checked was selected.  This resulted in the following lines trying to replace velocity with volume.